hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -172,7 +176,12 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
         <span className='padded-text'>{containers - missingContainersCount}/{containers}</span>
       </span>
     ) :
-      containers.toString();
+      <div>
+          <span>{containers.toString()}   </span>
+        <Tooltip placement='bottom' title={openContainers > 0 ? `Open Containers` : ``}>
```

Comment:
If we have 0 open containers we would still display it (eg. `3230 (0)`), so I think we can leave the `Open Containers` tooltip there regardless the number of open containers. 
```suggestion
        <Tooltip placement='bottom' title='Number of open containers'>
```

Reply:
Completed Review Changes.